autotest/ogr/ogr_mssqlspatial.py

```diff
@@ -711,6 +736,49 @@ def test_binary_field_bcp():
 #
 
 
+def test_geometry_column_identification():
+    """Test for issue GH #1190
+    SQL Server data source: geometry column not identified by ogr2ogr"""
+
+    if gdaltest.mssqlspatial_ds is None:
+        pytest.skip("MSSQLSpatial driver not available")
+
+    # once
+    try:
+        gdaltest.mssqlspatial_ds.ExecuteSQL("ALTER TABLE dbo.tpoly ADD shape GEOMETRY")
```

Comment:
that make this test step dependent on previous ones, and affect later ones.

Overall, looking at this test & ticket, I don't think it has anything to do with ogr2ogr. It is more about checking that ds.ExecuteSQL("SELECT ... the geometry column ...") returns a geometry column. No need for gdal.VectorTranslate() to check that

Reply:
done